test/Bridges/Constraint/indicator.jl

```diff
@@ -53,3 +53,92 @@ include("../utilities.jl")
         @test t.coefficient ≈ 1.0
     end
 end
+
```

Comment:
We should also have a test with `basic_constraint_test` (see https://github.com/JuliaOpt/MathOptInterface.jl/issues/750) and a test against `indicator1`, `indicator2` or `indicator3` (https://github.com/JuliaOpt/MathOptInterface.jl/blob/98a0e472e1757db14b849bbba8fe84597856def2/src/Test/intlinear.jl#L609-L611) with `test_model_equals` (see https://github.com/JuliaOpt/MathOptInterface.jl/issues/820)

Reply:
I didn't manage to make that work (yet)